src/commonNative/kotlin/com/epam/drill/jvmapi/Helper.kt

```diff
@@ -16,13 +16,24 @@
 package com.epam.drill.jvmapi
 
 import com.epam.drill.jvmapi.gen.*
-import com.epam.drill.logger.*
+import com.epam.drill.logger.api.*
 import kotlinx.cinterop.*
 import kotlin.native.concurrent.*
 import kotlin.reflect.*
 
-@SharedImmutable
-private val logger = Logging.logger("jvm api")
+
+private val loggerCallback_ = AtomicReference<(String) -> Logger?>({ _: String -> null }.freeze()).freeze()
+
+var loggerCallback: (String) -> Logger?
+    get() = loggerCallback_.value
+    set(value) {
+        loggerCallback_.value = value.freeze()
+    }
+
+private val logger_ = AtomicReference<Logger?>(null)
```

Comment:
```suggestion
private val _logger = AtomicReference<Logger?>(null)
```
Why is the underscore at the end after the variable name, and not before ?